### ISSUE TITLE
Support for column name (instead of number)

### ISSUE THREAD

**Issue by dwasyl** [vedmack/yadcf#616]
I'm trying to re-use the same (server-side) datatable code on a few different pages and ran into some issues with specifying column numbers. Being able to specify filters on the client side (and server side) with column names/data fields instead of order number would really enhance the flexibility of yadcf.

**Comment by vedmack** [comment]
I agree... but I don't see when I'll get the time to implement it in the near future, try implementing it yourself and send a PR

**Comment by dwasyl** [comment]
@vedmack I'm in the same boat, I've pulled up the source a few times to start looking into the changes required, but haven't had the time to dedicate. Happy to try leaving it on my list if no one else is working on it and if you don't have time for now.
